registry/handlers/manifests.go

```diff
@@ -82,7 +83,7 @@ type manifestHandler struct {
 
 // GetManifest fetches the image manifest from the storage backend, if it exists.
```

Comment:
nit: `image` also need to be removed from comments

Reply:
Removed.